BEST_PRACTICES.md

```diff
@@ -123,8 +136,8 @@ Image names should be named consistently to preserve organization and developer
 
 **For example:**
 
-* `refresh-bar-button-item` / `refresh-bar-button-item@2x` and `refresh-bar-button-item-selected` / `refresh-bar-button-item-selected@2x`
```

Comment:
I think we should avoid using image assets when possible and draw in code when possible. Much lighter and faster apps, less memory needed, less disk space, smaller downloads.